elastic/kibana
# [Ingest Pipelines] Add new two columns detail layout to pipeline details page

Closes https://github.com/elastic/kibana/issues/180769

## Summary

This PR improves the UX of the form fields in the pipeline details page by splitting the content into two columns and better separating the processors from the optional fields required for creating a pipeline. 

##### How to test

* Start up kibana and es
* Navigate to Stack management -> Ingest pipelines
* Create a new pipeline or view an existing pipeline
* Verify that:
  * Everything works as expected
  * Editing a pipeline name is still not allowed 
  
Flaky test runner: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/6226

https://github.com/elastic/kibana/assets/1191206/01b09bda-e80c-4c72-8e75-a3bda1c8d662
